src/helpers.coffee

```diff
@@ -49,8 +49,9 @@ extend = exports.extend = (object, properties) ->
 exports.flatten = flatten = (array) ->
   flattened = []
   for element in array
-    if '[object Array]' is Object::toString.call element
-      flattened = flattened.concat flatten element
+    # TODO: is this the same as using Object::toString.call?
```

Comment:
It is the same: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray?v=control#Polyfill

Let’s not change this function unless there’s a reason to.